src/config.rs

```diff
@@ -44,6 +40,13 @@ pub struct ListenerConfig {
     pub topic: String,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct PointCloud2ListenerConfig {
+    pub topic: String,
+    #[serde(default = "bool::default")]
+    pub use_rgb: bool,
```

Comment:
Unsure here. This flag is used to tell it to use the rgb field of the pointcloud. The default is a colorgradient based on z